ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -392,7 +392,7 @@ echo "$(date) - waiting for non-samples imagesteams to import..."
 count=0
 while :
 do
-  non_imported_imagestreams=$(oc -n openshift get imagestreams -o go-template='{{range .items}}{{$namespace := .metadata.namespace}}{{$name := .metadata.name}}{{range .status.tags}}{{if not .items}}{{$namespace}}/{{$name}}:{{.tag}}{{"\n"}}{{end}}{{end}}{{end}}' | (grep -v 'openjdk\|tomcat\|redhat-sso\|jboss' || true))
```

Comment:
why was tomcat removed?

Reply:
Not listed in https://bugzilla.redhat.com/show_bug.cgi?id=2106163#c1

Reply:
Pullspecs like `registry.redhat.io/jboss-webserver-3/webserver31-tomcat7-openshift:1.0` are still part of samples up [through 4.dev][1].  But I guess the `jboss` matcher would already have excepted them, so the `tomcat` entry here is redundant.

[1]: https://github.com/openshift/cluster-samples-operator/blob/1f21b78a35f05624351b29985f15cabda0d10e09/assets/operator/ocp-x86_64/webserver/imagestreams/jboss-webserver31-tomcat7-openshift-rhel7.json#L32